docs/get-started/vue3-vite.md

```diff
@@ -277,14 +277,35 @@ The definition above will generate the following controls:
 
 ### Limitations
 
-`vue-component-meta` cannot currently reference types from an import alias. You will need to replace any aliased imports with relative ones, as in the example below. See [this issue](https://github.com/vuejs/language-tools/issues/3896) for more information.
+If you're working with a project that relies on [`tsconfig references`](https://www.typescriptlang.org/docs/handbook/project-references.html) to link to other existing configuration files (e.g. `tsconfig.app.json`, `tsconfig.node.json`), we recommend that you update your [`.storybook/main.js|ts`](../configure/index.md) configuration file and add the following:
 
 ```ts
-// YourComponent.ts
-import type { MyProps } from '@/types'; // ❌ Cannot be resolved
-import type { MyProps } from '../types'; // ✅ Can be resolved
+// .storybook/main.ts
+import type { StorybookConfig } from '@storybook/vue3-vite';
+
+const config: StorybookConfig = {
+  framework: {
+    name: '@storybook/vue3-vite',
+    options: {
+      docgen: {
+        plugin: 'vue-component-meta',
+        tsconfig: 'tsconfig.app.json',
+      },
+    },
+  },
+};
+
+export default config;
 ```
 
+Otherwise, you might face missing component types/descriptions or unresolvable import aliases like `@/some/import`.
+
+<Callout variant="info">
+
+This is not a limitation of Storybook but instead from the `vue-component-meta` package. For more information, refer to the appropriate [GitHub issue](https://github.com/vuejs/language-tools/issues/3896).
+
+</Callout>
```

Comment:
@larsrickert small item here, when you're able, can you move the Callout above and place it after the code example?

Reply:
I changed it :) Did you mean it like that or did I get you wrong?

Reply:
@larsrickert all good, that's where it should be positioned. All that's left is to adjust the wording a bit based on @chakAs3 feedback so that we can get the correct information out. Sounds good?

Reply:
@chakAs3 Do you have a suggestion on how to rephrase this?

Reply:
"This is not a limitation of Storybook but rather a behavior stemming from the `vue-component-meta` package. For further details, please consult the relevant [GitHub issue](https://github.com/vuejs/language-tools/issues/3896)."